src/cli/createDidConfig.ts

```diff
@@ -12,17 +12,34 @@ import { DidConfigResource } from '../types'
 type KeyType = 'sr25519' | 'ed25519' | 'ecdsa'
 
 async function issueCredential(
-  keyUri: DidResourceUri,
+  key: string,
+  did: DidUri | undefined,
   origin: string,
   seed: string,
   keyType: KeyType,
   nodeAddress: string
 ) {
+  let didUri: DidUri
```

Comment:
I believe if the user doesn’t have to provide the assertion key ID we will be able to remove a lot of this complexity